UPGRADE-4.x.md

```diff
@@ -4,8 +4,9 @@ UPGRADE 4.x
 UPGRADE FROM 4.x to 4.x
 =======================
 
-### Upgrade to SonataDatagridBundle 3.0
+### Dependencies
 
+ - "sonata-project/datagrid-bundle" is bumped from ^2.4 to ^3.0.
```

Comment:
Just to give more context to my previous suggestion about this change. I'm proposing something like:
```md
### Dependencies

- "sonata-project/datagrid-bundle" is bumped from ^2.4 to ^3.0.
  Some description about the upgrade process for this dependency
  with all the required information.

- "package/a" is bumped from ^1.0 to ^2.0.
  Some description about the upgrade process for this dependency
  with all the required information.
```